_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
+author: "Maëlle Salmon, Andrew MacDonald, Kara Woo, Carl Boettiger, Jeff Hollister"
```

Comment:
Please see yaml in the checkers post for formatting (e.g. authors), use similar tags, etc:
https://github.com/ropensci/roweb/edit/master/_posts/2017-06-20-checkers.md